lib/web/mage/tabs.js

```diff
@@ -71,6 +71,9 @@ define([
                 anchorId = anchor.replace('#', '');
 
             if (anchor && isValid) {
+				if(anchorId == 'review-form'){
+                    anchorId = anchorId.replace('-form', 's');
+                }
                 $.each(self.contents, function (i) {
                     if ($(this).attr('id') === anchorId) {
```

Comment:
Hi @prakash2jcommerce, for better fix simply replace `if` condition in each with:
```
...
if ($(this).attr('id') === anchorId || $(this).find('#' + anchorId).length) {
...
```
and remove your fix.